packages/config/utils/utils.js

```diff
@@ -85,10 +132,13 @@ export const getContracts = (jsons, network) => {
  * @returns {boolean}
  */
 const hasPrivateKeys = flowJSON => {
-  return Object.entries(flowJSON?.accounts).reduce((hasPrivateKey, [key, value]) => {
-    if (hasPrivateKey) return true
-    return value?.hasOwnProperty("key")
-  }, false)
+  return Object.entries(flowJSON?.accounts).reduce(
+    (hasPrivateKey, [key, value]) => {
+      if (hasPrivateKey) return true
+      return value?.hasOwnProperty("key")
```

Comment:
@justinbarry Can we not just update this one line really quick to ignore the presence of `$` in the value field if key?